### DISCUSSION TITLE
Translatable blog

### DISCUSSION THREAD

**Discussion by comradekingu** [WeblateOrg/weblate#7527] [Polls]
A more unified website experience and
fixes making it into the source posts are possible,
This very much follows in the footsteps of examples in other areas, and stays true to "web-based continual libre localization".

I think this will help the public image of Weblate greatly.

Right now there are no translations, and no easy way to report or fix things.

@orangesunny The latest blogpost contains these errors: (Will delete once fixed)

Are you sure with your translations → Are you sure of your translations (or better, "Do the checks not apply to your source strings"?
Gittea → Should be Gitea
Pull request support → pull-request support
Not much was found broken this time → Not much was found to be broken this time around ("around" optional)
Rare condition related server timeouts. → Server timeouts triggered in rare conditions
or start a commercial project. → or start your own project. (there is no implication a libre project can't also be commercial. Weblate is for example both.)
Extended support subscribers → Extended-support subscribers, or "subscribers of the extended support plan(?)
list of Dedicated instance owners→ dedicated-instance owners, or "owners of dedicated instances".
ask our team anything Weblate → ask our team about anything Weblate.

**Comment by nijel** [comment]
There already is https://github.com/WeblateOrg/website/issues/84 :-)
